config/locales/emails_en.yml

```diff
@@ -67,5 +67,5 @@ en-GB:
           two: 2. The Central Office will let you know the outcome of your help with fees application, or if they need more information from you. This usually takes no longer than 5 working days.
       warning: 'Your case will not proceed until you send your help with fees reference number and employment tribunal claim number to the tribunal.'
     dwp_alert_notification:
-      subject: Help With Fees - DWP API Checker alert
+      subject: Help With Fees - DWP API Checker alert (%{environment})
```

Comment:
I assume this environment thing is OK to be nil ?  If someone hasn't set the env var, will things fail ?
If so, we need an example setting in a .env.example or something ?  otherwise where is the definitive list of env vars that we need to spin up the project ?